src/styles/styles.js

```diff
@@ -26,13 +26,11 @@ const picker = {
     fontFamily: fontFamily.EXP_NEUE,
     fontSize: variables.fontSizeNormal,
     lineHeight: variables.fontSizeNormalHeight,
-    paddingHorizontal: 11,
     paddingBottom: 8,
     paddingTop: 23,
+    paddingLeft: 0,
     height: variables.inputHeight,
     borderWidth: 0,
-    borderRadius: variables.componentBorderRadiusNormal,
-    textAlign: 'left',
```

Comment:
Curious why we need to remove this one?

Reply:
It was unnecessary. When I took it out there was no change, so I figured it was best to remove it.